v2/yarpcgrpc/outbound.go

```diff
@@ -147,8 +134,9 @@ func (o *Outbound) invoke(
 		callOptions...,
 	)
 
+	errInfo, appErrBody, err := invokeErrorToYARPCError(err, responseMD)
```

Comment:
Nit: maybe move this inline to 129, and have `invokeErrorToYARPCError(peer.clientConn.Invoke(...), ...)`. Makes it a little easier to see we don't forget the errorcheck after line 128.